Sources/Foundation/CMakeLists.txt

```diff
@@ -176,9 +176,19 @@ if(NOT BUILD_SHARED_LIBS)
   get_filename_component(icu_i18n_dir "${icui18n_path}" DIRECTORY)
   string(REPLACE "lib" "" icu_i18n_basename "${icu_i18n_basename}")
 
+  get_target_property(icuuc_path ICU::uc IMPORTED_LOCATION)
```

Comment:
cc @compnerd would these work wrt Windows?

Reply:
I believe that this should work fine with windows.  There is no `lib` prefix so it should be a no-op.  I believe this is already on main?